src/_data/sites/homeOfficeTricks.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://www.homeofficetricks.com/en/",
+	"name": "homeOfficeTricks",
+	"description": "Tipps and Tricks For The Best Homeoffice Possible",
+	"source_url": "https://github.com/dennishagemeier/homeofficetricks",
```

Comment:
@dennishagemeier This link 404s for me. Can you confirm that it's a public repo?
Or if it's a private repo, we can remove this optional field.